packages/lib-sourcify/src/lib/utils.ts

```diff
@@ -0,0 +1,42 @@
+declare const process: {
+  env: { [key: string]: string | undefined };
+};
+
+// Check if we are running in a Node.js environment
+export const isNode =
+  typeof process !== 'undefined' &&
+  // eslint-disable-next-line no-prototype-builtins
+  process.hasOwnProperty('env') &&
+  // eslint-disable-next-line no-prototype-builtins
+  process.env.hasOwnProperty('NODE_ENV');
+
+export const fs = isNode ? require('fs') : undefined;
+export const spawnSync = isNode
+  ? // eslint-disable-next-line @typescript-eslint/no-var-requires
+    require('child_process').spawnSync
+  : undefined;
+export const Path = isNode ? require('path') : undefined;
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+export const solc = isNode ? require('solc') : undefined;
+
+require('isomorphic-fetch');
```

Comment:
Should we keep this compatibility related code? I suppose yes if it's not breaking things

Reply:
These things are not necessary atm